### PR TITLE
define `cursor`, `order` and `limit` in AssetsRequestBuilder object.

### LINKED ISSUE #371: Feature Request: define order and limit in AssetsRequestBuilder object
### What problem does your feature solve?
When retrieving the list of assets it is not possible to se the `order` and `limit` in the **AssetsRequestBuilder** object which is used to perform the `Asset List` request. Developers must create instead a **RequestBuilder** to be able to pass these parameters

### What would you like to see?
Add `order(Order order)` and `limit(int number)` methods in the AssetsRequestBuilder

### What alternatives are there?
Developers can create and configure a RequestBuilder instead of working with  AssetsRequestBuilder abstraction

### PR DESCRIPTION
**Pull request by overcat**
Closes #371

### CODE REVIEW

File: src/main/java/org/stellar/sdk/requests/AssetsRequestBuilder.java [hunk 1]
```diff
@@ -39,4 +39,22 @@ public static Page<AssetResponse> execute(OkHttpClient httpClient, HttpUrl uri)
   public Page<AssetResponse> execute() throws IOException, TooManyRequestsException {
     return this.execute(this.httpClient, this.buildUri());
   }
+
+  @Override
```
Comment (sreuland) [inline src/main/java/org/stellar/sdk/requests/AssetsRequestBuilder.java:43]: looks good, can you add a test case in `AssetsRequestBuilderTest` to assert these new setters result in expected output from`AssetsRequestBuilder.buildUri()`  ?

Comment (overcat) [reply src/main/java/org/stellar/sdk/requests/AssetsRequestBuilder.java:43]: We now have such a test. https://github.com/stellar/java-stellar-sdk/blob/305974255130a2ebe42aeb289ed7040de8d75892/src/test/java/org/stellar/sdk/requests/AssetsRequestBuilderTest.java#L18-L20

**Review by sreuland — APPROVED** [review]

**Comment by overcat** [comment]
I just noticed that I mistakenly pushed the code from my other branch to this branch, so now the content of this PR has changed. I have updated the PR description. 🤦

**Comment by sreuland** [comment]
fortunately, this only merged to 'soroban' , should I revert the merge?

**Comment by overcat** [comment]
> fortunately, this only merged to 'soroban' , should I revert the merge?

Before, in order to release the soroban version more conveniently, the changes related to migrating to GitHub Actions were also added to the `soroban` branch. This PR depends on this feature, so I tend to merge this PR into the `soroban` branch. However, due to my mistake, the latest one commit message in soroban is incorrect. It should be "Generate separate stellar-sdk-javadoc.jar release artifact" instead of "define cursor, order and limit in AssetsRequestBuilder object". If you think it is necessary to correct it, we can revert this PR and then I will create a new one. 

Sorry for any inconvenience caused.

**Comment by overcat** [comment]
If this PR is reverted, then let's reopen #520.

**Comment by sreuland** [comment]
@overcat , since this is the last commit on `soroban` at present, we can avoid revert/redo, and instead I can `git commit --amend -m "Generate separate stellar-sdk-javadoc.jar release artifact(#519)" , it will remove the prior commit hash and generate a new one with same changeset and the new message and then I can force push that to `soroban`, is that ok to proceed?

**Comment by overcat** [comment]
> @overcat , since this is the last commit on `soroban` at present, we can avoid revert/redo, and instead I can `git commit --amend -m "Generate separate stellar-sdk-javadoc.jar release artifact(#519)" , it will remove the prior commit hash and generate a new one with same changeset and the new message and then I can force push that to `soroban`, is that ok to proceed?

This is acceptable, thank you.

**Comment by sreuland** [comment]
@overcat , i pushed the change, please pull latest from `soroban`, if you had any dev branches in-progress that were off soroban will need to rebase them.

**Pull request merged**